src/js/components/NameValueList/NameValueList.js

```diff
@@ -57,6 +57,7 @@ const NameValueList = forwardRef(
           ref={ref}
           columns={columns}
           gap={gap}
+          height="auto"
           fill={layout === 'grid'}
```

Comment:
```suggestion
```
I'm wondering if instead of adding `height='auto'` we could remove `fill={layout === 'grid'}`?

Reply:
good point,`height` is not needed as we can control height via fill prop. I think we have two options here

1) remove `fill={layout === 'grid'}` completely (we will remove height and width)
2) change fill to prop to `fill={layout === 'grid' && 'horizontal'}` (we will keep width: 100%)

Does grid width depend on the context and layout? Does width 100% is necessary or redundant? no idea :)
Let me know which option you prefer, thanks!

Reply:
From what I am seeing I don't think we need to have `width: 100%` so I think removing `fill` completely should be okay. Appreciate the thoughts and different options proposed though!